test/check-machines-create

```diff
@@ -547,9 +547,15 @@ vnc_password= "{vnc_passwd}"
                                                       start_vm=True))
 
     def testCreateUrlSource(self):
+        m = self.machine
```

Comment:
Since you added this variable here, I think it would be nice for it to be also used in rest of the function (rest of the function calls `self.machine.execute`), or not declare it at all and use self.machine.execute for workaround. Either way let's be consistent

Reply:
You are right, thanks! Fixed